tools/docker/tests/test_flows.sh

```diff
@@ -81,6 +81,14 @@ check_log() {
     check grep -q "$@" "${rootdir}/logs/${device}/beerocks_${program}.log"
 }
 
+send_bwl_event() {
+    # Send an event to a dummy bwl
+    # $1: device to send to (gateway, repeater1, repeater2)
+    # $2: radio to send to (wlan0, wlan2)
+    # $3: event string
+    check docker exec $1 sh -c 'echo '"$3"' > /tmp/$USER/beerocks/'$2'/EVENT'
```

Comment:
AFAIR, there is also an option to send Data event, so it is probably a good idea to add an option for that as well.
@arnout

Reply:
> AFAIR, there is also an option to send Data event, so it is probably a good idea to add an option for that as well.

They use the same file, they're just distinguished by the string `EVENT foo` vs. `DATA foo`.